ts/Stock/Indicators/IKH/IKHOptions.ts

```diff
@@ -49,19 +95,10 @@ export interface IKHGapSizeOptions {
     gapSize?: IKHOptions['gapSize'];
 }
 
-export interface IKHParamsOptions extends SMAParamsOptions {
-    periodTenkan?: number;
-    periodSenkouSpanB?: number;
-}
+/* *
+ *
+ *  Default Export
+ *
+ * */
 
-export interface IKHOptions extends SMAOptions {
-    chikouLine?: Record<string, CSSObject>;
-    gapSize?: number;
-    kijunLine?: Record<string, CSSObject>;
-    marker?: PointMarkerOptions;
-    params?: IKHParamsOptions;
-    senkouSpan?: IKHSenkouSpanOptions;
-    senkouSpanA?: Record<string, CSSObject>;
-    senkouSpanB?: Record<string, CSSObject>;
-    tenkanLine?: Record<string, CSSObject>;
-}
+export default IKHOptions;
```

Comment:
Is this default export needed for anything? Named export works and this type is not imported anywhere using default export.